disabled_apps/clock/manifest.webmanifest

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Clock",
+  "description": "Gaia Clock",
+  "lang": "en",
+  "short_name": "Clock",
+  "scope": "./",
+  "start_url": "./index.html?homescreen=1",
+  "launch_path": "/index.html",
+  "developer": {
+    "name": "The Gaia Team",
+    "url": "https://github.com/mozilla-b2g/gaia"
+  },
+  "default_locale": "en-US",
+  "icons": [{
+    "sizes": "84x84",
+    "src": "./style/icons/clock_84.png"
```

Comment:
I think this should be "style/icons/clock_84.png", relative to the directory the manifest is in. But I could be wrong.